apps/docs/src/app/platform/component-docs/platform-display-list-item/platform-display-list-item-docs.component.ts

```diff
@@ -3,10 +3,10 @@ import { Component } from '@angular/core';
 import { ExampleFile } from '../../../documentation/core-helpers/code-example/example-file';
 
 import * as dliSrc from '!raw-loader!./platform-display-list-item-examples/platform-display-list-item-example.component.html';
-import * as borderLessDLISrc from '!raw-loader!./platform-display-list-item-examples/platform-borderless-display-list-item-example.component.html';
-import * as borderLessDLITs from '!raw-loader!./platform-display-list-item-examples/platform-display-list-item-border-less-example.component.ts';
+import * as borderLessDLISrc from '!raw-loader!./platform-display-list-item-examples/platform-display-list-item-border-less-example.component.html';
+import * as borderLessDLITs from '!raw-loader!./platform-display-list-item-examples/platform-display-list-item-border-less-example.component';
```

Comment:
just wondering if this will work in stackblitz without `.ts` extension?

Reply:
yes, I checked :slightly_smiling_face: